testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/page/DeleteAccountPage.java

```diff
@@ -0,0 +1,29 @@
+package org.keycloak.testsuite.ui.account2.page;
+
+import javax.ws.rs.core.UriBuilder;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import static org.keycloak.testsuite.util.UIUtils.clickLink;
+
+/**
+ * @author Zakaria Amine <zakaria.amine88@gmail.com>
+ */
+public class DeleteAccountPage extends AbstractLoggedInPage  {
+
+  @Override
+  public String getPageId() {
+    return "delete-account";
+  }
+
+  @Override
+  public boolean isCurrent() {
+    return driver.getTitle().contains("Keycloak Account Management") && driver.getCurrentUrl().split("\\?")[0].endsWith("/account/#/delete-account");
```

Comment:
Please just use `URLUtils.currentUrlEquals(toString());` which is much simpler and easier to use.

Reply:
since now moving to the functionality to Account page, I will be deleting this class.